plugins/modules/primary_ip.py

```diff
@@ -127,6 +154,21 @@
             sample:
                 key: value
                 mylabel: 123
+        assignee_id:
+            description: ID of the resource the Primary IP is assigned to, null if it is not assigned.
+            type: int
+            returned: always
+            sample: 1937415
+        assignee_type:
+            description: Resource type the Primary IP can be assigned to.
```

Comment:
```suggestion
            description: Resource type the Primary IP is assigned to.
```

Reply:
From docs:

Reply:
I believe the documentation has a few mistakes, I'll discuss this internally with the responsible team.

Reply:
You are correct, I had a bad understanding of the description of this field.